discojs-web/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@epfml/discojs",
+  "name": "@epfml/discojs-web",
```

Comment:
Does it make sense to name it this way? @epfml/discojs-web currently doesn't exist on npm

Reply:
it does now :)
we discussed it in #621 and from what I remember, we wanted to follow the naming practice from tfjs, having a base import of `@epfml/discojs` and platform specifics as `@epfml/discojs-{node,web}`.